js/script.js

```diff
@@ -22,16 +23,213 @@ const greenquadrant = document.querySelector("#greenquad");
+// Click the power button to activate the game
 
 powerOn.addEventListener('click', (event) => {
   if (powerOn.checked == true) {
-    power = true
-    countTurn.innerHTML = "_";
+    powerSelection = true
+    countTurn.innerHTML = "--";
+    startButton.disabled = false;
+    strictOn.disabled = false
 
   }else {
-    power = false
+    powerSelection = false
+    startButton.disabled = true;
+    strictOn.disabled = true;
     countTurn.innerHTML = "";
+    // clearFlash();
+    clearInterval(intervalId);
+    strictOn.checked = false;
+  }
+});
+
+// Click the start button to start the game
+startButton.addEventListener('click', (event) => {
+  if (powerSelection == true) {
+    playNewGame();
+    }
+});
+
+
+// Enable strict button to start strict mode.
+strictOn.addEventListener('click',(event) => {
+  if(strictOn.checked == true){
+    strictMode = true;
+    strictPlay();
+  }
+});
+
+
+// Function to play the game
+function playNewGame () {
+// reset variables
+  winTheGame = false;
+  userSeq = [];
+  sound = true;
+  intervalId = 0;
+  quadFlash = 0;
+  playTurn = 1;
+  countTurn.innerHTML = 1;
+  goodPlay = true;
+  startGame = true;
+  compPlay();
+
+};
+
+function click(num){
+  const [audio, quad, lite] = getAttributes(num)
+  if(sound){
+    audio.play()
+  }
+  quad.classList.add(lite)
+  setTimeout(function(){
+    quad.classList.remove(lite)
+  }, 500)
+}
+
+function getAttributes(num){
+  switch (num) {
+    case 1:
+      return [audio1, greenquadrant, "greenlite"]
+    case 2:
+      return [audio2, redquadrant, "redlite"]
+    case 3:
+      return [audio3, bluequadrant, "bluelite"]
+    case 4:
+      return [audio4, yellowquadrant, "yellowlite"]
+  }
+}
+
+function compPlay() {
+  countTurn.innerHTML = round;
+  powerSelection = true;
+  winTheGame = false;
+
+  if(compSeq.length < round) {
+    compSeq.push(Math.floor(Math.random() * 4) + 1);
+  }
+
+  // Generate a set of random button presses
+  for (let i = 0; i < compSeq.length; i++) {
+    setTimeout(function(){
+      let currentPlay = compSeq[i]
+      click(currentPlay)
+    }, i * 1000)
+  }
+}
+
+
+// User play function
+function userplay(quads) {
+  var userTurn = userSeq.length
+
+  if (userTurn < round) {
+
+    if (compSeq[userTurn] == quads && round == 20) {
+      counter.innerHTML == "WIN"
+    }
+      userSeq.push(quads);
+  }
+   var userResult = true;
+   if(userSeq.length == round) {
+     for (var i = 0; i < round ; i++) {
+       if (userSeq[i] != compSeq[i]){
+         userResult = false;
+         break;
+       }
+     }
+
+     if (userResult){
+       userSeq = [];
+       round ++
+
+       setTimeout(function() { compPlay();}, 1000);
+     }else {
+       userSeq = [];
+     }
+   }
+}
+
+
+// sound created with each quad pad press
+var audio1 = new Audio();
+audio1.src = "https://s3.amazonaws.com/freecodecamp/simonSound1.mp3";
+greenquad.addEventListener('click', (event) => {
+  if (startGame && powerSelection){
+    clickGreen();
```

Comment:
we no longer need these, just call click with the number of the quad. We should also improve these addEventListeners by adding them in a loop because we are writing the exact same code to add the listeners. We want to keep our code `DRY` - Don't Repeat Yourself.